src/modules/UI/scenes/Otp/OtpSettingsSceneComponent.js

```diff
@@ -138,6 +130,13 @@ export default class OtpSettingsScene extends Component<OtpSettingsSceneProps, S
           </View>
           {this.renderModals(styles)}
         </View>
+        <StaticModalComponent
+          cancel={this.cancelStatic}
+          body={this.state.messageModalMessage}
+          bodyComponent={this.state.messageModalComponent}
+          isVisible={this.state.showMessageModal}
+          modalDismissTimerSeconds={8}
+        />
```

Comment:
@amanalili I am not sure about this change. The reason there were two if statements is that different modals were rendered at different times. By putting them in the same if statement and return, you are always going to render the TwoButton Modal, and then the Static Modal on top of it. You may not be showing it because of the isVisible property change. The reason for separate return statements was so we would only render the modal when it is needed and not when it is not. ... Please explain the reason for this change.

Reply:
I tested the app, with this scene and it works, so my concern may not be valid.

Reply:
Using an always visible modal is un natural (or not standard) way of using modals. Also the styling of the modals is unnatural (like no background color) because it is always visible and it is controlled adding/removing the component itself. If you are having concern of modals appearing at the same time then I suggest a clearer states that falsify the state of one modal when another one is truthy.